docs/testing/README.md

```diff
@@ -60,38 +60,44 @@ Here is an example of testing an orchestrator, but ingesting test data. This can
 ```python
 from spetlrtools.testing.TestHandle import TestHandle
 from spetlr.utils import DataframeCreator
+from spetlrtools.testing import DataframeTestCase
+from pyspark.sql.types import StructType
 
-test_schema =  ...
-test_columns = ...
-test_data = ...
 
-expected_schema =  ...
-expected_columns = ...
-expected_data = ...
+class ExampleETLTest(DataframeTestCase):
+    def test_etl(self):
 
-params = BusinessETLParameters()
+        test_schema =  StructType([...])
+        test_columns = [...]
+        test_data = [...]
 
-params.source_dh =  TestHandle(
-            provides=DataframeCreator.make_partial(
-                test_schema , 
-                test_columns,
-                 test_data 
-            )
-        )
+        expected_schema =  StructType([...])
+        expected_columns = [...]
+        expected_data = [...]
 
-params.target_dh =  TestHandle(
-            provides=DataframeCreator.make_partial(
-                expected_schema , 
-                expected_columns,
-                [] 
-            )
-        )
+        params = BusinessETLParameters()
+
+        params.source_dh =  TestHandle(
+                    provides=DataframeCreator.make_partial(
+                        test_schema, 
+                        test_columns,
+                        test_data 
+                    )
+                )
+
+        params.target_dh =  TestHandle(
+                    provides=DataframeCreator.make_partial(
+                        expected_schema, 
+                        expected_columns,
+                        expected_data 
+                    )
+                )
 
-orchestrator= BusinessETLOrchestrator(params)
+        orchestrator= BusinessETLOrchestrator(params)
```

Comment:
Nitty gritty but add a missing space so that `orchestrator= Business...` -> `orchestrator = Business...`. Pro tip, whenever you write python in a markdown file quickly add it to a dummy python-file and use black to format it :)

Reply:
Good idea!